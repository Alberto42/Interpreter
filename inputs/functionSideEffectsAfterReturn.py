a = 0
def f(x) {
    return 0
    a = 1
}

f(0)
print(a)