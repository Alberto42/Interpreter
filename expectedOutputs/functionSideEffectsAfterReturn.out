Ident "a" = 0

0

